.github/settings.yml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2021 Cargill Incorporated
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+repository:
+  # See https://developer.github.com/v3/repos/#edit for all available settings.
+  name: sawtooth-rfcs
+  homepage: https://wiki.hyperledger.org/display/sawtooth
+  private: false
+  has_issues: true
+  has_projects: false
+  has_wiki: false
+  has_downloads: false
+  default_branch: main
```

Comment:
are we going to change these repos from master to main first?

Reply:
Merging the PRs should update the default branch automatically. :)